zcbor/zcbor.py

```diff
@@ -981,11 +981,11 @@ def __init__(self, *args, **kwargs):
     # Name of variables and enum members for this element.
     def var_name(self, with_prefix=False):
         if with_prefix or not self.short_names:
-            name = ("_%s" % self.id(with_prefix=with_prefix))
+            name = ("%s" % self.id(with_prefix=with_prefix))
         else:
             name = self.id(with_prefix=with_prefix)
             if name in ["int", "bool", "float"]:
-                name = "_" + name
+                name = name.capitalize()
```

Comment:
Why do we capitalize?

Reply:
It's a way to avoid naming collision since "int", "bool", and "float" are already taken.